docs/lang/articles/reference.md

```diff
@@ -250,8 +250,89 @@ with `expression2` as the error message.
 
 ## Compound statements
 
+Compound statements contain (groups of) other statements;
+they affect or control the execution of those other statements in some way.
+In general, compound statements span multiple lines,
+although in simple incarnations a whole compound statement may be contained in one line.
```

Comment:
I know this is from https://docs.python.org/3/reference/compound_stmts.html#compound-statements, but I don't get why incarnations is used here (a sincere question). To me it seems this is just "in simple situations". cc @Vissidarte-Herman